ddtrace/sampler.py

```diff
@@ -52,7 +62,8 @@ class RateSampler(BaseSampler):
     It samples randomly, its main purpose is to reduce the instrumentation footprint.
     """
 
-    def __init__(self, sample_rate=1):
+    def __init__(self, sample_rate=1.0):
+        # type: (float) -> None
         if sample_rate < 0:
             raise ValueError("sample_rate of {} is negative".format(sample_rate))
         elif sample_rate > 1:
```

Comment:
This needs to go away 🤦‍♂️

Reply:
Do you mean the type hint or changing the default value to be a float?

Reply:
lol nothing for this PR, just me cringing at this code that needs to change